src/CommitmentsV2/SFA.DAS.CommitmentsV2.Api/Extensions/IWebHostBuilderExtensions.cs

```diff
@@ -13,7 +9,9 @@ public static class IWebHostBuilderExtensions
         public static IWebHostBuilder ConfigureDasAppConfiguration(this IWebHostBuilder hostBuilder)
         {
             return hostBuilder.ConfigureAppConfiguration(c => c
-                .AddAzureTableStorage(CommitmentsConfigurationKeys.CommitmentsV2));
+                .AddAzureTableStorage(CommitmentsConfigurationKeys.CommitmentsV2)
+                .AddAzureTableStorage(CommitmentsConfigurationKeys.EncodingConfiguration)
```

Comment:
It's probably more efficient to pass multiple keys to `AddAzureTableStorage` rather than registering the provider twice e.g. `AddAzureTableStorage(CommitmentsConfigurationKeys.CommitmentsV2, CommitmentsConfigurationKeys.EncodingConfiguration)`.